tetris.js

```diff
@@ -203,6 +175,34 @@ module.exports = {
 		}
 
 	},
+
+	checkThatTheFieldIsFree: function (arrayOfCoordinates, field) {
+		if (!this.allItemsOnTheField(arrayOfCoordinates, field)) {
+			return false
+		}
+		for (let coordinate of arrayOfCoordinates) {
+			if (field[coordinate.y][coordinate.x]) {
+				return false
+			}
+		}
+		return true;
+	},
+
+	allItemsOnTheField: function (arrayOfCoordinates, field) {
+		let lengthField_X = field[0].length;
+		let lengthField_Y = field.length;
+		for (let coordinate of arrayOfCoordinates) {
+			if (
+				(coordinate.x < 0 || coordinate.y < 0) ||
+				(coordinate.x >= lengthField_X) ||
+				(coordinate.y >= lengthField_Y)
```

Comment:
join this line and the next
this is confusing to leave if statement round brackets by its own